curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md

```diff
@@ -29,7 +29,7 @@ const ourMusic = [
 ];
 ```
 
-This is an array which contains one object inside. The object has various pieces of <dfn>metadata</dfn> about an album. It also has a nested `formats` array. If you want to add more album records, you can do this by adding records to the top level array. Objects hold data in a property, which has a key-value format. In the example above, `"artist": "Daft Punk"` is a property that has a key of `artist` and a value of `Daft Punk`. [JavaScript Object Notation](http://www.json.org/) or `JSON` is a related data interchange format used to store data.
+This is an array which contains one object inside. The object has various pieces of <dfn>metadata</dfn> about an album. It also has a nested `formats` array. If you want to add more album records, you can do this by adding records to the top level array. Objects hold data in a property, which has a key-value format. In the example above, `"artist": "Daft Punk"` is a property that has a key of `artist` and a value of `Daft Punk`. JavaScript Object Notation or `JSON` lightweight data-interchange format. It is easy for humans to read and write. It is easy for machines to parse and generate.
```

Comment:
Honestly, I think the text about JSON is unrelated to this challenge. Personally, I think everything from "JavaScriptObject Notation or `JSON`..." down to line 46 can be deleted.

But if we keep it, I would keep it simple and just say something like "JavaScript Object Notation or `JSON` lightweight data-interchange format for storing objects as a string. Here is the JSON for this  object:"

Reply:
this was what I originally planned, lets keep this for a day. if no one against it, I will go forward with deleting it. otherwise I will alter the text.

Reply:
I agree, we should just remove the bit about JSON.